README.mdown

```diff
@@ -53,6 +53,17 @@ php composer.phar install
 
 Once the dependencies are installed you should configure the repositories you want to review.
 
+## Using AWS
+
+To use aws, you can use [packer](http://packer.io) to generate an Amazon Machine Image to run the application
```

Comment:
What about using `https://www.packer.io/`? That way there's no redirect for anyone that clicks on that.